lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -593,3 +609,33 @@ def build_response_answers(response: Union[str, List]) -> List:
         return create_questionnaire_response_item(question_id, answers)
 
     return None
+
+
+def create_follow_up_encounter(record: REDCapRecord, patient_reference: dict, site_reference: dict) -> tuple:
+    """
+    Returns a FHIR Encounter resource entry and reference for a follow-up
+    encounter
+    """
+    encounter_identifier = create_identifier(
+        system = f"{INTERNAL_SYSTEM}/encounter",
+        value = f"{REDCAP_URL}/{record.project.id}/{record['record_id']}_follow_up"
+    )
+    encounter_class_coding = create_coding(
+        system = "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+        code = "HH"
+    )
+
+    if not record.get('fu_timestamp'):
+        return None, None
+
```

Comment:
This check can be done at the top of the `create_follow_up_encounter` function before creating an `encounter_identifier` or `encounter_class_coding`.